backend/src/apps/slack/models/workspace.py

```
@@ -126,10 +147,20 @@ def invite_link_alert_threshold(self) -> int | None:
         )
         return self.invite_link_member_count + offset
 
+    @property
+    def is_content_reporting_enabled(self) -> bool:
+        """Return True when content reporting is configured for this workspace."""
+        return bool((self.content_report_alert_channel_id or "").strip())
+
     def __str__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)